File: src/codes/Users_Manual/modulation/wav_1.cpp
//~#include<autd3.hpp>
//~#include "autd3/modulation/audio_file.hpp"
//~int main() {
//~const auto path = "path/to/foo.wav";
const autd3::modulation::audio_file::Wav m(
    path, 4u * autd3::kHz,
    autd3::modulation::SincInterpolation(autd3::modulation::BlackMan{32}));
//~return 0; }

File: src/codes/Users_Manual/modulation/wav_1.cs
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Modulation.AudioFile;
~using static AUTD3Sharp.Units;
~var path = "path/to/foo.wav";
var m = new Wav(path, 4u * kHz, new SincInterpolation(new BlackMan(32)));